cmd/gcp-controller-manager/node_csr_approver.go

```diff
@@ -1007,6 +1019,63 @@ func deleteAllPodsBoundToNode(ctx *controllerContext, nodeName string) error {
 	return utilerrors.NewAggregate(errs)
 }
 
+func markFailedAndDeletePodWithCondition(ctx *controllerContext, pod *v1.Pod, condition *corev1apply.PodConditionApplyConfiguration) error {
+	fieldManager := "GCPControllerManager"
```

Comment:
have it as a constant that can be used in `WithMessage` above.

Reply:
Done